docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+    }
+
+    function play() external payable {
+        uint256 randomNumber = random();
+
+        if(randomNumber % 2 == 0){
+            minter.mintNativeCoin(
+                msg.sender,
+                msg.value * 2
+            );
+        }
+    }
+}
+```
+
+When you're done, it should look like this:
+
+![files](.github/images/files.png)
+
+### Compiling and deploying
+
+After creating the files, we should compile `Game.sol`.
+
+![Compile](.github/images/compiling.png)
+
+Go to the deploy tab, choose injected web3 as enviroment - connect your wallet - then choose Game in the contract dropdown. Then click the deploy button.
+
+![Deploy](.github/images/deploying.png)
+
+Confirm the transaction with your metamask wallet.
+
+![Deployed contracts](.github/images/game_deployed.png)
+
+Yay! We deployed a smart contract in our Subnet! That sounds fantastic!
+
+## Setting Game Contract as a Minter
+
+We're admin in [`contractNativeMinterConfig`](#contractnativeminterconfig).
+Which means we can add new Minters to the network using `NativeMinterInterface`.
+
+We already created `NativeMinterInterface.sol`
+
+Go to remix and compile `NativeMinterInterface.sol` by following the same steps as `Game.sol`.
+
+Go to the deploy tab in remix, and choose NativeMinterInterface in the contract dropdown. Then type in the address `0x0200000000000000000000000000000000000001` to `Load contract from Address` then click `At Address`
+
+![Interface steps](.github/images/interface.png)
+
+After clicking the button, you should see the `NATIVEMINTERINTERFACE AT 0X02..` under the `Deployed Contracts` section.
+
+We will call the `setEnabled` function of `NativeMinter` contract with the address of `Game` contract to set the `Game` contract as a minter.
+
+![Set minter](.github/images/setenabled.png)
+
+Confirm the transaction with your metamask wallet. We can call this function because we've added our address as an admin in the `contractNativeMinterConfig` section.
+
+Now the `Game.sol` contract is a minter and can mint native tokens!
+
+## Play the Game
+
+In the deploy tab: 
+- Set the value to 100 Ether or any other amount you want to play with.
+- Find `Game` contract under deployed contracts section in remix.
+- Call `play` function.
+- Confirm the transaction with your metamask wallet.
+
+![play](.github/images/play.png)
+
+Then check your metamask wallet.
+
+![result](.github/images/lucky.png)
+
+I first played with 10 and then 100 tokens. Seems like it's my lucky day! I already earned 110 LUCK tokens. As you can see, our smart contract **can mint native tokens!**
+
+**Let's go all in!**
+
+![allin](.github/images/outofluck.png)
+
+I'm out of **luck**!
+
+As an admin in `NativeMinter` contract, I can mint native tokens as well! Let's mint ourselves 1000 LUCK tokens.
+
+![minting native tokens](.github/images/mintingtomyself.png)
+
+And my luck is back!
+
+![luck back](.github/images/luckisback.png)
+
+# Conclusion
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) You shouldn't have multiple `<h1>`s in the same document.

> MD025/single-title/single-h1: Multiple top-level headings in the same document

[More info: MD025](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document)